backend/btrixcloud/profiles.py

```diff
@@ -442,6 +442,17 @@ async def add_profile_file_replica(
             {"$push": {"resource.replicas": {"name": ref.name, "custom": ref.custom}}},
         )
 
+    async def delete_org_profiles(self, oid: UUID):
```

Comment:
This will not delete the profile data in storage which is deleted with individual `delete_profile()` calls.

Reply:
Reverting